flag-app/flag-app-project/src/hooks/useApiData.ts

```diff
@@ -3,11 +3,15 @@ import { useQuery } from "react-query";
 import { Country } from "../interfaces";
 import { apiClient } from "../services";
 
-export const useApiData = (searchedQuery?: string) => {
-  const endpoint = searchedQuery ? `/name/${searchedQuery}` : "/all";
+export const useApiData = (searchedQuery?: string, filteredQuery?: string) => {
```

Comment:
therefore these types must be changed to `(searchedQuery: string, filteredQuery: string)`